examples/flows/standard/describe-image/question_on_image.jinja2

```diff
@@ -0,0 +1,6 @@
+# system:
```

Comment:
This prompt seems too general. Can we reference the default prompt used for OpenAI GPT-4V tool ? Like this:

# system:
As an AI assistant, your task involves interpreting images and responding to questions about the image.
Remember to provide accurate answers based on the information present in the image.

# user:
{{question}}
![image]({test_image}})

Reply:
Changed.